src/ducks/transactions/actions/ReimbursementStatusAction/Card.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react'
+import UICard from 'cozy-ui/react/Card'
+import { Caption, Text } from 'cozy-ui/react/Text'
+import Icon, { iconPropType } from 'cozy-ui/react/Icon'
+import cx from 'classnames'
+import styles from 'ducks/transactions/actions/ReimbursementStatusAction/Card.styl'
+
+// TODO use icons from cozy-ui when https://github.com/cozy/cozy-ui/pull/983
+// has been merged
+import iconPhone from 'assets/icons/icon-phone.svg'
+import iconEmail from 'assets/icons/icon-email.svg'
+
+const Card = props => {
+  const { title, caption, icon, className, ...rest } = props
+
+  return (
+    <UICard className={cx(styles.Card, className)} {...rest}>
+      <Icon icon={icon} size={16} color="var(--coolGrey)" />
+      <div className="u-ml-1">
+        <Text>{title}</Text>
+        <Caption>{caption}</Caption>
+      </div>
+    </UICard>
+  )
+}
+
+Card.propTypes = {
+  icon: iconPropType.isRequired,
+  title: PropTypes.string.isRequired,
+  caption: PropTypes.node.isRequired
+}
+
+export const DumbPhoneCard = props => {
+  const { contact, t, ...rest } = props
+  const href = `tel:${contact.number}`
+  const caption = contact.price ? (
+    <>
+      {contact.price}
+      {t('ReimbursementStatusModal.contact.phoneCallPrice')}
+    </>
+  ) : (
+    t('ReimbursementStatusModal.contact.phoneCallFree')
+  )
+
+  return (
+    <Card
+      href={href}
+      title={contact.number}
+      caption={caption}
+      icon={iconPhone}
+      tag="a"
+      {...rest}
+    />
+  )
+}
+
+DumbPhoneCard.propTypes = {
+  contact: PropTypes.shape({
+    price: PropTypes.string,
+    number: PropTypes.string.isRequired
+  }).isRequired,
+  t: PropTypes.func.isRequired
+}
+
+export const PhoneCard = translate()(DumbPhoneCard)
+
+export const DumbWebCard = props => {
+  const { contact, t, ...rest } = props
+  const caption = new URL(contact.href).hostname
+
+  return (
+    <Card
+      href={contact.href}
+      title={t(`ReimbursementStatusModal.contact.actions.${contact.action}`)}
+      caption={caption}
+      icon={iconEmail}
+      target="_blank"
+      tag="a"
+      {...rest}
+    />
+  )
+}
+
+DumbWebCard.propTypes = {
+  contact: PropTypes.shape({
+    href: PropTypes.string.isRequired,
+    action: PropTypes.string.isRequired
+  }).isRequired,
```

Comment:
Nit: There are two contact objects that seem to be different. It might be a bit confusing.

Maybe naming the `webContact = PropType.shape({ ... })` would make it clearer.